internal/api/external.go

```diff
@@ -222,29 +212,11 @@ func (a *API) internalExternalProviderCallback(w http.ResponseWriter, r *http.Re
 		return nil
 	})
 
-	rurl := a.getExternalRedirectURL(r)
 	if err != nil {
 		return err
 	}
 
-	if errEmailVerificationRequiredOrConfirmationSent != nil {
-		flowType := models.ImplicitFlow
-		if flowState != nil {
-			flowType = models.PKCEFlow
-		}
-		msg := fmt.Sprintf("Unverified email with %v. ", providerType)
-		if errors.Is(errEmailVerificationRequiredOrConfirmationSent, errEmailVerificationRequired) {
-			msg += fmt.Sprintf("Verify the email with %v in order to sign in", providerType)
-		} else if errors.Is(errEmailVerificationRequiredOrConfirmationSent, errEmailConfirmationSent) {
-			msg += fmt.Sprintf("A confirmation email has been sent to your %v email", providerType)
-		}
-		rurl, err = a.prepErrorRedirectURL(unauthorizedError(msg), w, r, rurl, flowType)
```

Comment:
@J0 i don't think we need to use `prepErrorRedirectURL` here because we already wrap the entire function in `redirectErrors()` - https://github.com/supabase/gotrue/blob/94293b72b829436308050be5399069976b810cdb/internal/api/external.go#L140

Reply:
decided to keep the hash fragments since it could be used by folks who either set [`detectSessionInUrl`](https://github.com/supabase/gotrue-js/blob/c9296bbc27a2f036af55c1f33fca5930704bd021/src/GoTrueClient.ts#L90) to false or those who opt to hit gotrue's API directly and handle parsing the error from the response returned